.github/workflows/docker-build-push.yml

```
@@ -14,7 +14,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Log in to Docker Hub
-        uses: docker/login-action@v2.1.0
+        uses: docker/login-action@v2.2.0
```

Do we care about precise pins? README just uses v2.

```suggestion
        uses: docker/login-action@v2
```